src/http-adapter.js

```diff
@@ -76,32 +80,39 @@ class HttpAdapter {
    * @returns {Promise} Promise for destroy.
    */
   destroy (link) {
-    return this._ajax('DELETE', link)
+    return this._http(METHOD.DELETE, link)
   }
 
-  _ajax (type, url, data) {
-    let headers = {
-      'Accept': 'application/vnd.api+json',
-      'Content-Type': 'application/vnd.api+json'
+  async _http (method = '', url = '', data = {}) {
+    let init = {
+      method,
+      headers: {
+        'Accept': 'application/vnd.api+json',
+        'Content-Type': 'application/vnd.api+json'
+      }
```

Comment:
I don't really see the advantage of `init` over individual identifiers that get packed into the final call to `fetch`...

Reply:
I can refactor it . Just want to make sure I understand; In other words, just pass method into the final call?

Reply:
Yes, and `headers`. They're more descriptive than `init`.